docker-compose.yaml

```diff
@@ -61,6 +61,7 @@ x-airflow-common: &airflow-common
     AIRFLOW__CORE__DAGS_ARE_PAUSED_AT_CREATION: "true"
     AIRFLOW__CORE__LOAD_EXAMPLES: "false"
     AIRFLOW__API__AUTH_BACKENDS: "airflow.api.auth.backend.basic_auth,airflow.api.auth.backend.session"
+    # AIRFLOW__WEBSERVER__RELOAD_ON_PLUGIN_CHANGE: "true" # used when modifying plugins
```

Comment:
why remove this one? It's useful when you modify plugins

Reply:
you are right, although we should add set it to false in our kubernetes as it may cause high memory usage and race conditions:
https://airflow.apache.org/docs/apache-airflow/stable/configurations-ref.html#reload-on-plugin-change